.github/workflows/CI.yml

```diff
@@ -7,6 +7,9 @@ jobs:
       matrix:
         os: [ ubuntu-latest ]
         node-version: [ 14 ]
+        integration-deps:
+        - diagram-js@7.x bpmn-js@8.x
+        - diagram-js@8.x bpmn-js@9.0.0-alpha.2
```

Comment:
Should we make this optional in the happy path?

I.e. only install `integration-deps` in parts of our CI build and test against everything specified in `package.json` as a happy path?

Reply:
Please explain what you mean by "happy path" in this context. Can you give me an example of desired outcome?

Reply:
Exactly what Martin refers to, too:

https://github.com/bpmn-io/bpmn-js-properties-panel/pull/581#discussion_r802666568

There should be a configuration where we do not install these additional dependencies, but simply test things as specified in the `package.json`.